app/components/PhotoUploadStatus/index.js

```diff
@@ -0,0 +1,126 @@
+//@flow
+import React, { Fragment, type ComponentType } from 'react';
+import Card from 'app/components/Card';
+import Icon from 'app/components/Icon';
+import Button from 'app/components/Button';
+import Flex from 'app/components/Layout/Flex';
+import { connect } from 'react-redux';
+import { Gallery } from 'app/actions/ActionTypes';
+import Tooltip from 'app/components/Tooltip';
+import {
+  selectGalleryPictureById,
+  type UploadStatus,
+  type GalleryPictureEntity,
+  initialUploadStatus
+} from 'app/reducers/galleryPictures';
+
+type StateProps = {
+  uploadStatus: UploadStatus,
+  lastImage: ?GalleryPictureEntity
+};
+
+type DispatchedProps = {
+  hideUploadStatus: () => void
+};
+
+type Props = StateProps & DispatchedProps;
+
+const style = {
+  zIndex: 2,
+  margin: 40,
+  position: 'fixed',
+  bottom: 0,
+  left: 0
+};
+const UploadStatusCard = ({
+  uploadStatus,
+  lastImage,
+  hideUploadStatus
+}: Props) => {
+  if (!uploadStatus.showStatus) return null;
+
+  const uploadDone =
+    uploadStatus.successCount + uploadStatus.failCount ===
+    uploadStatus.imageCount;
+
+  const hasFailedUploads = uploadStatus.failCount !== 0;
+
+  return (
+    <Card style={style}>
+      <Button onClick={hideUploadStatus}>
+        <Icon name="close" />
+      </Button>
```

Comment:
This button is both kinda ugly and could rather easily be interpreted as "Cancel upload". That's what I thought it was at first. Just having the Icon without the button and moving it to the right corner would probably help with both problems.

Reply:
Yeah. Lazyness it is :laughing: